backend/app/Models/Todo.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Todo extends Model
+{
+    protected $fillable = ['title', 'status'];
+
+    /**
+     * 状態定義
+     */
+    const STATUS = [
+        1 => ['label' => '未着手', 'class' => 'bg-danger'],
+        2 => ['label' => '着手中', 'class' => 'bg-info'],
+        3 => ['label' => '完了', 'class' => 'bg-success'],
+    ];
+
+    /**
+     * 状態のラベル
+     * @return string
+     */
+    public function getStatusLabelAttribute()
+    {
+        // 状態値
+        $status = $this->attributes['status'];
+
+        // 定義されていなければ空文字を返す
+        if (!isset(self::STATUS[$status])) {
```

Comment:
👍

Reply:
[こまかいはなし]

`if !hoge` みたいなコードは読むときに、 「もしhogeじゃないとき」
というような感じになるので少し読み手に思考させる負荷がかかります。
今回はまあすぐ分かると思うので良いのですがｗ
例えば `if (!hoge && !fuga)` のように増えていったりすると「じゃないかつじゃない」みたいな感じになってわかりづらくなっていきます。
なので

```suggestion
        if (empty(self::STATUS[$status]))
```

このように「空だったら」というような感じで"否定"を入れないような形にしていくと読み手の負荷も減るので
少し意識しておくと良いかなと思います！

Reply:
なるほど！確かに否定表現が入るとどっち？ってなるので、
その辺も意識した方が良いですね！